server/pulp/server/db/manage.py

```diff
@@ -274,3 +290,13 @@ def _start_logging():
     console_handler = logging.StreamHandler()
     console_handler.setLevel(logging.INFO)
     _logger.root.addHandler(console_handler)
+
+
+def _user_input_continue(question):
+    reply = str(raw_input(_(question + ' (y/n): '))).lower().strip()
```

Comment:
Can this default to no? I think the prompt would become `(y/N`)` What do you think about this?